Library/Homebrew/global.rb

```diff
@@ -30,7 +30,7 @@
 HOMEBREW_USER_AGENT_CURL = ENV["HOMEBREW_USER_AGENT_CURL"]
 HOMEBREW_USER_AGENT_RUBY = "#{ENV["HOMEBREW_USER_AGENT"]} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
```

Comment:
You mean the “Waiting on Bintray” code? Yes, I think it should be using this, but nobody noticed the omission thus far.